__tests__/app.test.js

```diff
@@ -1,40 +1,115 @@
+const testData = require("../db/data/test-data");
+const request = require("supertest");
 beforeEach(() => runSeed(testData));
-describe('GET non-existent endpoint', () => {
-    test("returns 404", () => {
-        return request(app)
-        .get('/api/cagetories')
-        .expect(404);
-    });
-    test("returns error message", () => {
-        return request(app)
-        .get('/api/test')
-        .then((res) => {
-            expect(res.body).toEqual({ msg: "error: invalid path" });
-        })
-    })
-})
-describe('GET /api/categories', () => {
-    test("returns status code 200", () => {
-        return request(app)
-        .get('/api/categories')
-        .expect(200);
-    });
-    test("returns array of objects", () => {
-        return request(app)
-        .get('/api/categories')
-        .then((res) => {
-            expect(Array.isArray(res.body.categories)).toBe(true);
-        })
-    })
-    test("returns correct response", () => {
-        return request(app)
-        .get('/api/categories')
-        .then((res) => {
-            expect(res.body.categories).toMatchObject(testData.categoryData);
-        })
-    })
-})
\ No newline at end of file
+describe("GET non-existent endpoint", () => {
+  test("returns 404", () => {
+    return request(app).get("/api/cagetories").expect(404);
+  });
+  test("returns error message", () => {
+    return request(app)
+      .get("/api/test")
+      .then((res) => {
+        expect(res.body).toEqual({ msg: "error: invalid path" });
+      });
+  });
+});
+describe("GET /api/categories", () => {
+  test("returns status code 200", () => {
+    return request(app).get("/api/categories").expect(200);
+  });
+  test("returns array of objects", () => {
+    return request(app)
+      .get("/api/categories")
+      .then((res) => {
+        expect(Array.isArray(res.body.categories)).toBe(true);
+      });
+  });
+  test("returns correct response", () => {
+    return request(app)
+      .get("/api/categories")
+      .then((res) => {
+        expect(res.body.categories).toMatchObject(testData.categoryData);
+      });
+  });
+});
+describe("GET /api/reviews", () => {
+  test("returns status code 200", () => {
+    return request(app).get("/api/reviews").expect(200);
+  });
+  test("returns an array", () => {
+    return request(app)
+      .get("/api/reviews")
+      .then((res) => {
+        expect(Array.isArray(res.body.reviews)).toBe(true);
+      });
+  });
+  test("returns array of correct length", () => {
+    return request(app)
+      .get("/api/reviews")
+      .then((res) => {
+        expect(res.body.reviews.length).toBe(testData.reviewData.length);
+      });
+  });
+  test("returns objects with expected properties", () => {
+    return request(app)
+      .get("/api/reviews")
+      .then((res) => {
+        const props = ["title", "owner", "category", "review_img_url", "designer", "review_id", "votes", "comment_count", "created_at"]
+        for (let review of res.body.reviews) {
```

Comment:
might have been easier to forEach through the array, and check the keys with .toMatchObject